src/app-logic/constants.js

```diff
@@ -54,3 +54,38 @@ export const TRACK_VISUAL_PROGRESS_LINE_WIDTH = 2;
 
 // Height of the active tab resources panel header.
 export const ACTIVE_TAB_TIMELINE_RESOURCES_HEADER_HEIGHT = 20;
+
+// ═════════════════════════════════════════════════════════════════════════════
+// Storage and server-related constants
+// ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+// For the 2 values GOOGLE_STORAGE_BUCKET and PROFILER_SERVER_ORIGIN, several
+// values are possible, so that you can easily switch between existing server
+// (both local or remote).
+//
+// GOOGLE_STORAGE_BUCKET
+// ⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺⎺
```

Comment:
Ah that's weird, the font that github uses can't draw these ones with the same width as other characters.

Reply:
Yeah, I've found some fixed-width fonts have trouble with uncommon character. I'd prefer to keep them to more standard characters, but I'm also not hugely opinionated here as long as we have docs.